### PR TITLE
added login cli interface and publish big fixed

### PR DESCRIPTION
**Pull request by dhillondeep**

### CODE REVIEW

File: cmd/wio/commands/pac/user/login.go [hunk 1]
```diff
@@ -50,5 +65,7 @@ func (cmd Login) Execute() error {
         return err
     }
     log.WriteSuccess()
+    log.Infoln(log.Yellow, "user logged in")
```
Comment (Mogball) [inline cmd/wio/commands/pac/user/login.go]: capitalize User?

File: cmd/wio/toolchain/npm/client/client.go [hunk 2]
```diff
@@ -31,10 +31,11 @@ func GetJson(client *http.Client, req *http.Request, target interface{}) (int, e
         return 0, err
     }
     status := resp.StatusCode
+    err = json.NewDecoder(resp.Body).Decode(target)
     if status != http.StatusOK && status != http.StatusCreated {
         return status, nil
```
Comment (Mogball) [inline cmd/wio/toolchain/npm/client/client.go]: can probably return the error here too

**Review by Mogball — CHANGES_REQUESTED** [review]

File: cmd/wio/commands/pac/user/login.go (hunk 1 above)
Comment (dhillondeep) [reply cmd/wio/commands/pac/user/login.go]: true

**Comment by dhillondeep** [comment]
@Mogball made changes

**Pull request merged**